common/djangoapps/student/tests/test_email.py

```diff
@@ -94,6 +97,8 @@ class ActivationEmailTests(TestCase):
 
     def setUp(self):
         super(ActivationEmailTests, self).setUp()
+        self.inactive_user = UserFactory(is_active=False)
```

Comment:
`setUp` gets run with every test method in the suite, so if you're only using the inactive user once (same goes for the `RequestFactory` below), just use it in the test itself